inference-engine/tests/functional/plugin/shared/src/single_layer_tests/activation.cpp

```diff
@@ -39,10 +40,15 @@ std::string ActivationLayerTest::getTestCaseName(const testing::TestParamInfo<ac
 void ActivationLayerTest::SetUp() {
     InferenceEngine::Precision netPrecision;
     std::pair<std::vector<size_t>, std::vector<size_t>> shapes;
-    std::tie(activationType, netPrecision, shapes, targetDevice) = GetParam();
+    std::pair<ngraph::helpers::ActivationTypes, std::vector<float>> activationDecl;
+    std::tie(activationDecl, netPrecision, shapes, targetDevice) = GetParam();
+
+    activationType = activationDecl.first;
```

Comment:
Minor comment, we can do without `activationType` and `constantsValue`

Reply:
@iefode Yes. But maybe it would be more understandable and readable?